iothub/service/src/Http/HttpTransportSettings.cs

```diff
@@ -25,7 +25,7 @@ public HttpTransportSettings()
         /// An instance of a class that implements <see cref="IWebProxy"/>.
         /// </value>
         /// <remarks>
-        /// The <see cref="HttpTransportSettings"/> class is only used for the <see cref="JobClient"/> or the <see cref="RegistryManager"/>; so the proxy set here will only be valid for those clients.
+        /// The <see cref="HttpTransportSettings"/> class is only used for the <see cref="ScheduledJobsClient"/> or the <see cref="RegistryManager"/>; so the proxy set here will only be valid for those clients.
```

Comment:
@timtay-microsoft looks like we have an old reference to `RegistryManager` here.